Cargo.toml

```
@@ -3,8 +3,8 @@ name = "range-map"
 version = "0.2.0"
 authors = ["Joe Neeman <joeneeman@gmail.com>"]
 description = "Maps and sets implemented using ranges."
-documentation = "http://jneem.github.io/range-map"
-repository = "http://github.com/jneem/range-map"
+documentation = "https://jneem.github.io/range-map"
```

```suggestion
documentation = "https://docs.rs/range-map/latest/range_map/"
```

While we're at it, let's point the docs link at `docs.rs`. I think this crate might predate docs.rs, which is why it was pointing elsewhere...